lib/manageiq/providers/ibm_cloud/cloud_tool.rb

```diff
@@ -55,6 +55,10 @@ def vpc(region: 'us-east', version: '2021-01-01', generation: 2)
           @vpc ||= CloudTools::Vpc.new(:cloudtools => self, :region => region, :version => version, :generation => generation)
         end
 
+        def events(region: 'us-south', service_key:)
```

Comment:
We should probably default to the same region as vpc above: `def vpc(region: 'us-east',` as long as the service is available in that region